Parsers/ASimDns/Product parsers/vimDnsInfobloxNIOS.yaml

```diff
@@ -0,0 +1,125 @@
+Parser:
+  Title: InfoBlox
+  Version: '0.0'
+  LastUpdated: July 7, 2021
+Product:
+  Name: Infoblox
+Normalization:
+  Schema: Dns
+  Version: '0.2.0'
+References:
+- Title: ASIM DNS Schema
+  Link: https://aka.ms/AzSentinelDnsDoc
+- Title: ASIM
+  Link: https://aka.ms/AzSentinelNormalization
+Description: |
+  This is a Query Parser that is used to map Infoblox NIOX Dns Events (Syslog) to the Azure Sentinel Information Model Dns schema.
+ParserName: vimDnsInfobloxNIOS
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr
+    Type: string
+    Default: '*'
+  - Name: domain_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: responsecodename
+    Type: string
+    Default: '*'
+  - Name: dnsresponsename
+    Type: string
+    Default: '*'
+  - Name: response_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype
+    Type: string
+    Default: 'lookup'
+ParserQuery: |
+  let Infoblox=(
+    starttime:datetime=datetime(null), endtime:datetime=datetime(null)
+    , srcipaddr:string="*"
+    , domain_has_any:dynamic=dynamic([]) 
+    , responsecodename:string="*", dnsresponsename:string="*"
+    , response_has_any:dynamic=dynamic([]) , eventtype:string="lookup"
+    ){
+    let RawData = Syslog 
+    // *****************************************************************
+    //  Pre-parsing filterring:
+    // This filtering is not accurate. Further filtering down the line
+        | where
+          (isnull(starttime) or TimeGenerated > starttime)
+          and (isnull(endtime) or TimeGenerated < endtime) 
+          and (srcipaddr=="*" or  has_ipv4( SyslogMessage ,srcipaddr))
+          and (array_length(domain_has_any) ==0 or SyslogMessage has_any (domain_has_any) )
+          and (array_length(domain_has_any) ==0 or SyslogMessage has_any (domain_has_any))
+          and (responsecodename=="*" or SyslogMessage has responsecodename)
```

Comment:
What about responsename (i.e. response_has_any_ipv4)?

Reply:
also...not available